spec/game_spec.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+
+require_relative '../lib/game.rb'
+
+RSpec.describe WinGame do
+  let(:cell) { [[1, 2, 3], [4, 5, 6], [7, 8, 9]] }
+  let(:result) { WinGame.new(cell) }
+
+  describe '#result.position' do
+    it 'returns the result of the position of the element in the board' do
+      expect(result.cell).to eql(cell)
+    end
+  end
+
+  describe 'possible positon to win on the first row' do
+    let(:horizontal) { [%w[x x x], [4, 5, 6], [7, 8, 9]] }
+    let(:result_h) { WinGame.new(horizontal) }
+    it 'returns true when the first row has 3 equal symbols on the horizontal' do
+      expect(result_h.win_game).to eql(true)
+    end
+  end
+
+  describe 'possible positon to win on the second row' do
+    let(:horizontal) { [[1, 2, 3], %w[x x x], [7, 8, 9]] }
+    let(:result_h) { WinGame.new(horizontal) }
+    it 'returns true when the second row has 3 equal symbols on the horizontal' do
+      expect(result_h.win_game).to eql(true)
+    end
+  end
+
+  describe 'possible positon to win on the third row' do
+    let(:horizontal) { [[1, 2, 3], [4, 5, 6], %w[x x x]] }
+    let(:result_h) { WinGame.new(horizontal) }
```

Comment:
Let's move this `let` block outside the `describe` block to where `let(:result) { WinGame.new(cell) }` is in Line 7, this will help prevent declaring it several times. 

Please let's do this for other `let` matchers that are been repeated.
- `let(:result_h) { WinGame.new(horizontal) }`
- `let(:result_v) { WinGame.new(vertical) }`
- `let(:result_d) { WinGame.new(diagonal) }`

> Read about [DRY rule](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and use it.